apps/server/src/modules/common-cartridge/common-cartridge-client/lesson-client/mapper/lesson-dto.mapper.spec.ts

```diff
@@ -183,7 +181,7 @@ describe('LessonDtoMapper', () => {
 		describe('when mapping LessonResponse to lesson DTO with lernstore content', () => {
 			const setup = () => {
 				const lessonContentResponse: LessonContentResponse = {
-					content: { resources: [faker.internet.url(), faker.lorem.text()] } as ComponentLernstorePropsImpl,
+					content: { resources: [faker.internet.url(), faker.lorem.text()] } as unknown as ComponentLernstorePropsImpl,

```

Comment:
Here 'unknown' must be used because the types do not match (it is now an object array, not string array). Matching types without 'unknown' would be better, like this:

```suggestion
					content: {
						resources: [{ url: faker.internet.url(), description: faker.lorem.text() }],
					} as ComponentLernstorePropsImpl,
```